<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2025-01-31 14:35:15.344 ***</b>
=ERROR REPORT==== 31-Jan-2025::14:35:15.344441 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.247127.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1975}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1936}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2025-01-31 14:35:15.344 ***</b>
=CRASH REPORT==== 31-Jan-2025::14:35:15.344767 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.247132.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2428)
    ancestors: [<0.247127.0>]
    message_queue_len: 0
    messages: []
    links: [<0.247127.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 987
    stack_size: 29
    reductions: 594
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2025-01-31 14:35:55.671 ***</b>
=ERROR REPORT==== 31-Jan-2025::14:35:55.670995 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.251478.0&gt;,
                               {#Ref&lt;0.141372340.535560193.231322&gt;,1003881,ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.141372340.535560193.231322&gt;,1003881,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2025-01-31 14:35:55.671 ***</b>
=CRASH REPORT==== 31-Jan-2025::14:35:55.671266 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.251499.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.141372340.535560193.231322&gt;,1003881,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [&lt;0.251478.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 414
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2025-01-31 14:40:38.824 ***</b>
=CRASH REPORT==== 31-Jan-2025::14:40:38.824736 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.257318.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.141372340.537133057.156689&gt;,7531,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1696)
    ancestors: [&lt;0.257316.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28123
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-31 14:40:48.231 ***</b>
=ERROR REPORT==== 31-Jan-2025::14:40:48.230009 ===
** Generic server <0.258590.0> terminating 
** Last message in was {nodedown,'peer_down_crash-453954-11901@a615e64a8a7c'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-453954-11901",
                              peer_down => crash},
                            'peer_down_crash-453954-11901@a615e64a8a7c',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,50,46,49,47,98,
                             105,110,47,101,114,108],
                            ["-sname","peer_down_crash-453954-11901",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQGE2MTVlNjRhOGE3YwAD8h4AAAAAZ5zO/A=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-453954-11901@a615e64a8a7c'}},
                            {<0.258585.0>,#Ref<0.141372340.537133057.170794>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-453954-11901@a615e64a8a7c'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2025-01-31 14:40:48.231 ***</b>
=CRASH REPORT==== 31-Jan-2025::14:40:48.231261 ===
  crasher:
    initial call: peer:init/1
    pid: <0.258590.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-453954-11901@a615e64a8a7c'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2478)
    ancestors: [<0.258585.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1497
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-31 14:40:49.080 ***</b>
=WARNING REPORT==== 31-Jan-2025::14:40:48.078660 ===
Process <0.92.0> on node 'peer_SUITE-shutdown_stop-373092-11901@a615e64a8a7c' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-31 14:40:49.577 ***</b>
=WARNING REPORT==== 31-Jan-2025::14:40:49.577380 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-415397-11901@a615e64a8a7c'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-31 14:40:49.776 ***</b>
=WARNING REPORT==== 31-Jan-2025::14:40:49.776362 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-511299-11901@a615e64a8a7c'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-01-31 14:41:05.436 ***</b>
=WARNING REPORT==== 31-Jan-2025::14:41:04.434512 ===
Process <0.89.0> on node 'peer_SUITE-shutdown_stop-415877-11901@a615e64a8a7c' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-01-31 14:41:05.843 ***</b>
=WARNING REPORT==== 31-Jan-2025::14:41:05.843400 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-455170-11901@a615e64a8a7c'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-01-31 14:41:05.878 ***</b>
=WARNING REPORT==== 31-Jan-2025::14:41:05.878292 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-373348-11901@a615e64a8a7c'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-31 14:41:08.108 ***</b>
=ERROR REPORT==== 31-Jan-2025::14:41:08.108565 ===
** Generic server <0.258967.0> terminating 
** Last message in was {'EXIT',<0.258951.0>,
                               {#Ref<0.141372340.537133057.173953>,1566563,ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-511619-11901",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-511619-11901@a615e64a8a7c',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,50,46,49,47,98,105,110,47,101,114,
                                     108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-511619-11901",
                                     "-user","peer"],
                                    #Port<0.3135>,undefined,<<>>,running,
                                    {<0.258951.0>,
                                     #Ref<0.141372340.529793026.122398>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.141372340.537133057.173953>,1566563,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-31 14:41:08.109 ***</b>
=CRASH REPORT==== 31-Jan-2025::14:41:08.108884 ===
  crasher:
    initial call: peer:init/1
    pid: <0.258967.0>
    registered_name: []
    exception exit: {#Ref<0.141372340.537133057.173953>,1566563,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.258951.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3135>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4658
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-31 14:41:08.682 ***</b>
=ERROR REPORT==== 31-Jan-2025::14:41:08.681703 ===
** Generic server <0.258992.0> terminating 
** Last message in was {'EXIT',<0.258952.0>,
                           {#Ref<0.141372340.529006596.69995>,2153884,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "MDWBWUJMSJREVIWQKKCZ","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@a615e64a8a7c.2025-01-31_14.24.11/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-511491-11901",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@a615e64a8a7c.2025-01-31_14.24.11/make_test_dir.stdlib_test.logs/run.2025-01-31_14.24.47/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-511491-11901",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5.7/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-511491-11901@a615e64a8a7c',
                                    "/usr/local/lib/erlang-26.2.5.7/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-511491-11901",
                                     "-setcookie","MDWBWUJMSJREVIWQKKCZ",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@a615e64a8a7c.2025-01-31_14.24.11/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-511491-11901",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@a615e64a8a7c.2025-01-31_14.24.11/make_test_dir.stdlib_test.logs/run.2025-01-31_14.24.47/log_private/",
                                     "-user","peer"],
                                    #Port<0.3157>,undefined,<<>>,running,
                                    {<0.258952.0>,
                                     #Ref<0.141372340.537133057.174492>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.141372340.529006596.69995>,2153884,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-31 14:41:08.682 ***</b>
=CRASH REPORT==== 31-Jan-2025::14:41:08.682177 ===
  crasher:
    initial call: peer:init/1
    pid: <0.258992.0>
    registered_name: []
    exception exit: {#Ref<0.141372340.529006596.69995>,2153884,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.258952.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3157>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4331
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-31 14:41:09.061 ***</b>
=WARNING REPORT==== 31-Jan-2025::14:41:08.060089 ===
Process <0.89.0> on node 'peer_SUITE-shutdown_stop-511587-11901@a615e64a8a7c' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-31 14:41:09.246 ***</b>
=WARNING REPORT==== 31-Jan-2025::14:41:09.246442 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-373988-11901@a615e64a8a7c'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-31 14:41:09.498 ***</b>
=WARNING REPORT==== 31-Jan-2025::14:41:09.498402 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-374084-11901@a615e64a8a7c'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-31 14:41:09.555 ***</b>
=WARNING REPORT==== 31-Jan-2025::14:41:09.555404 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-511683-11901@a615e64a8a7c'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-31 15:02:10.292 ***</b>
=ERROR REPORT==== 31-Jan-2025::15:02:10.291789 ===
** Generic server <0.335009.0> terminating 
** Last message in was {'EXIT',<0.335007.0>,
                               {#Ref<0.141372340.567803905.161072>,351140,
                                [{child1,<0.335016.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.335009.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.335015.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458474],
                            1,0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.141372340.567803905.161072>,351140,
    [{child1,<0.335016.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-31 15:02:10.292 ***</b>
=ERROR REPORT==== 31-Jan-2025::15:02:10.291789 ===
** Generic server <0.335010.0> terminating 
** Last message in was {'EXIT',<0.335007.0>,
                               {#Ref<0.141372340.567803905.161072>,351140,
                                [{child1,<0.335016.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.335010.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.335016.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458474],
                            1,0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.141372340.567803905.161072>,351140,
    [{child1,<0.335016.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-31 15:02:10.292 ***</b>
=CRASH REPORT==== 31-Jan-2025::15:02:10.292098 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.335009.0>
    registered_name: []
    exception exit: {#Ref<0.141372340.567803905.161072>,351140,
                     [{child1,<0.335016.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.335007.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1010
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-31 15:02:10.292 ***</b>
=CRASH REPORT==== 31-Jan-2025::15:02:10.292254 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.335010.0>
    registered_name: []
    exception exit: {#Ref<0.141372340.567803905.161072>,351140,
                     [{child1,<0.335016.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.335007.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1027
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_bridge_SUITE:simple_global_supervisor/1 2025-01-31 15:02:53.768 ***</b>
=CRASH REPORT==== 31-Jan-2025::15:02:53.768529 ===
  crasher:
    initial call: supervisor_bridge:supervisor_bridge_SUITE/1
    pid: <0.435179.0>
    registered_name: []
    exception exit: killed
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [bridge9212,<0.435174.0>]
    message_queue_len: 0
    messages: []
    links: [<0.435180.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 388
  neighbours:
    neighbour:
      pid: <0.435180.0>
      registered_name: work_supervisor_bridge_SUITE_server
      initial_call: {supervisor_bridge_SUITE,internal_loop_init,1}
      current_function: {supervisor_bridge_SUITE,internal_loop,1}
      ancestors: []
      message_queue_len: 0
      links: [<0.435179.0>]
      trap_exit: false
      status: waiting
      heap_size: 233
      stack_size: 3
      reductions: 9
      current_stacktrace: [{supervisor_bridge_SUITE,internal_loop,1,
                      [{file,"supervisor_bridge_SUITE.erl"},{line,152}]}]

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during zzz_SUITE:lc_graph/1 2025-01-31 15:28:26.758 ***</b>
=WARNING REPORT==== 31-Jan-2025::15:28:26.755161 ===
Process <0.438632.0> on node 'test_server@a615e64a8a7c' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2025 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Fri Jan 31 2025 14:24:47<!--/date--><br />
</div>
</center>
</body>
</html>
